andela/titan-dev-ah-frontend
# #164611280 Add categories on article

#### What does this PR do?
- add categories on 
landing page using HTML and CSS
#### Description of Task to be completed?
- Create the HTML and CSS page
- Make it responsive for both mobile and desktop
#### How should this be manually tested?
- Fetch origin
- Check out the branch ft-add-categories-on-article-164611280
- Open the following file pages/landing.html in chrome or firefox
- And use the inspector to view the page with different devices.
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
 - #164611280
#### Screenshots (if appropriate):
- Desktop Version
<img width="1439" alt="Screen Shot 2019-03-14 at 16 08 55" src="https://user-images.githubusercontent.com/25322304/54363503-b5fdc000-4673-11e9-9374-80ccaf5e0bc6.png">
- Mobile version
<img width="362" alt="Screen Shot 2019-03-14 at 16 09 53" src="https://user-images.githubusercontent.com/25322304/54363510-ba29dd80-4673-11e9-9e09-2b9e46c46f82.png">


#### Questions:
None
